data/remnant/remnant logistics.txt

```diff
@@ -0,0 +1,295 @@
+		"reactor vessels"
+		"welding and fabrication robots"
+		"air purification systems"
+		"water purificaton systems"
+		"environmental controls"
+		"heating and cooling systems"
+		"xeno biologic reactor vessels"
+		"catalyst reactors"
+		"imaging and integrity scanners"
+		"leak detection equipment"
+		"liquid stream mixers"
+		"quality assurance lab equipment"
+		"catalytic reagents"
+		"reference isotopes"
+		"chemical feedstocks"
+		"material handling equipment"
+		"liquid and gas stream dryers"
+		"heat exchangers"
+		"turbines"
+		"stainless steel piping"
+		"complex alloy piping"
+		"radioactive decontamination kits"
+		"hazmat suits"
+		"oil/chemical spill containment units"
+		"mechanical mixers and agitators"
+		"sintered metal filters"
+		"industrial gases"
+		"ultrasonic agitators"
+
+mission "Remnant: Logistics Industrial Delivery"
+	name "Industrial Equipment Delivery"
+	description `Project teams on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo  "remnant industrial" 30 300
+	job
+	repeat
+	to offer
+			not "Remnant: Ssil Vida Active"
+			not "flagship planet: Ssil Vida"
+			has "Remnant: Logistics 1: done"
+			has "Remnant: Logistics Civil Engineering: done" >= 9
+			random < 76
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project technician inspects the <cargo> and approves your payment of <payment>.`
+		payment 55000 15
+
+trade	
+	commodity "remnant outfitter repair"
+		"Inhibitor Cannon"
+		"Thrasher Cannon"
+		"Thrasher Turret"
+		"Point Defense Turret"
+		"EMP Torpedo Storage Bay"
+		"EMP Torpedo Bay"
+		"Millennium Cell"
+		"Epoch Cell"
+		"Aeon Cell"
+		"Thermoelectric Cooler"
+		"Emergency Ramscoop"
+		"Research Laboratory"
+		"Salvage Scanner"
+		"Anvil-Class Engine"
+		"Crucible-Class Thruster"
+		"Forge-Class Thruster"
+		"Smelter-Class Thruster"
+		"Crucible-Class Steering"
+		"Forge-Class Steering"
+		"Smelter-Class Steering"
+
+mission "Remnant: Logistics Outfitter Delivery"
+	name "Outfitter Equipment Delivery"
+	description `Outfitter repair crews on <planet> need <cargo> parts delivered. Logistics adjustment is <payment>.`
+	cargo "remnant outfitter repair" 30 175
+	job
+	repeat
+	to offer
+			not "Remnant: Ssil Vida Active"
+			not "flagship planet: Ssil Vida"
+			has "Remnant: Logistics 1: done"
+			has "license: Remnant"
+			has "Remnant: Logistics Industrial Delivery: done" >= 9
+			random < 30
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A repair technician inspects the <cargo> parts and authorizes your payment of <payment>.`
+		payment 60000 20
+
+trade
+	commodity "remnant salvage"
+		"Builder salvage from the Graveyard"
```

Comment:
```suggestion
		"salvaged Builder"
```
`30 tons of Builder salvage from the Graveyard components` -> `30 tons of salvaged Builder components`